backend/src/controllers/movies.js

```diff
@@ -14,7 +25,9 @@ const getByCategory = (req, res, next) => {
 };
 
 const getDetails = (req, res, next) => {
-  const { id } = req.params;
+  const {
```

Comment:
Why did we require this change? did we have some problems with req/params being undefined?.
As a suggestion we can use lodash get for this if we had the undefined issue, something like:
```suggestion
  const id = get("params.id", req);
```

Reply:
Actually this was an agreement that we did with @Rolly007 in order to unify the code, but I really was curious to use lodas/fp since I almost never used it, I applied the changes in order to practice